### PR TITLE
Testing alternative sound implementation (WIP) [skip ci]

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

File: source/glest_game/global/core_data.h [hunk 1]
```diff
@@ -357,23 +356,15 @@ namespace Glest
...
-            static const string sfxMarker;
-            static const string sfxMenuClickA;
-            static const string sfxMenuClickB;
-            static const string sfxMenuClickC;
-            PlaySoundClip();
-            ~PlaySoundClip();
+    StaticSound *getSound (const string& iniPlaySoundVal);
 
-      };
+    StaticSound sfxAttention;
```
Comment (andy5995) [inline source/glest_game/global/core_data.h]: This is causing multiple definition errors

**Comment by Jammyjamjamman** [comment]
So the problem right now is creating the `StaticSound` objects. A segmentation fault occurs when calling `Config.getInstance()`. Something necessary probably isn't loaded yet before I try to load the StaticSounds.

**Comment by Jammyjamjamman** [comment]
Update: I think I will discontinue this branch until I get more insight into why I can't load sounds at this point in the code.

**Comment by andy5995** [comment]
I'm sorry I lack the knowledge to help more with it.

**Comment by Jammyjamjamman** [comment]
No problem... I can't see it either xD.
I think this helps describe my current problem a bit better here: https://github.com/Jammyjamjamman/zetaglest-source/commit/503e765057f6a60733f6f7a2774ef2dd2522bdc7

**Comment by mathusummut** [comment]
Should we close this pull request?

**Comment by Jammyjamjamman** [comment]
@mathusummut 
>Should we close this pull request?

Yes

**Pull request closed without merging**